verifier/src/tcb.rs

```diff
@@ -278,10 +280,79 @@ impl<'a> TryFrom<&'a str> for TcbInfoRaw<'a> {
     }
 }
 
+/// Verifier for ensuring a raw TCB was signed with the provided key
```

Comment:
I kind of think that the `TcbInfoRawVerifier` is all that should be in this crate, and there should be a separate crate which contains the other TCB types.
Not sure though, and also not sure what to name it. Part of the reason is this file is getting a bit big and another part is that most of the other modules are verifiers only with the types being verified living elsewhere.

At the same time I think we can get by this way for now.